nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/rs2d/RS2DData.java

```diff
@@ -149,6 +149,7 @@ public Dataset toDataset(String datasetName) throws IOException {
                 false, ByteOrder.BIG_ENDIAN, 0);
         dataset.newHeader();
         for (int i = 0; i < nDim; i++) {
+            resetSW(i);
```

Comment:
Tested loading the input directory processed datasets, and JRES had an incorrect SW, this fixed it, and didn't break loading any of the other datasets in the test set. There is a todo in this file for loading datasets which may be why the SW wasn't correct but wasn't sure the reasoning behind the todo so didn't make any changes for it.
https://github.com/nanalysis/nmrfx/blob/2b397999fa056f8977e215a835618e181fb2c6f7/nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/rs2d/RS2DData.java#L264